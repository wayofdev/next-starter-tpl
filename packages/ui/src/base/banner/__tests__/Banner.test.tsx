import renderer from 'react-test-renderer'
import { Banner } from '../Banner'

it('should match snapshot', () => {
  const tree = renderer
    .create(<Banner message="Big news! We're excited to announce a brand new product." />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
