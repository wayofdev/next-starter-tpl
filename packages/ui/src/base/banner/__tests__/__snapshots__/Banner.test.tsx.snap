// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div
  className="bg-indigo-600"
>
  <div
    className="mx-auto max-w-7xl p-3 sm:px-6 lg:px-8"
  >
    <div
      className="flex flex-wrap items-center justify-between"
    >
      <div
        className="flex w-0 flex-1 items-center"
      >
        <span
          className="flex rounded-lg bg-indigo-800 p-2"
        >
          <svg
            aria-hidden="true"
            className="h-6 w-6 text-white"
            fill="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M16.881 4.346A23.112 23.112 0 018.25 6H7.5a5.25 5.25 0 00-.88 10.427 21.593 21.593 0 001.378 3.94c.464 1.004 1.674 1.32 2.582.796l.657-.379c.88-.508 1.165-1.592.772-2.468a17.116 17.116 0 01-.628-1.607c1.918.258 3.76.75 5.5 1.446A21.727 21.727 0 0018 11.25c0-2.413-.393-4.735-1.119-6.904zM18.26 3.74a23.22 23.22 0 011.24 7.51 23.22 23.22 0 01-1.24 7.51c-.055.161-.111.322-.17.482a.75.75 0 101.409.516 24.555 24.555 0 001.415-6.43 2.992 2.992 0 00.836-2.078c0-.806-.319-1.54-.836-2.078a24.65 24.65 0 00-1.415-6.43.75.75 0 10-1.409.516c.059.16.116.321.17.483z"
            />
          </svg>
        </span>
        <p
          className="ml-3 truncate font-medium text-white"
        >
          <span
            className="md:hidden"
          >
            We announced a new product!
          </span>
          <span
            className="hidden md:inline"
          >
            Big news! We're excited to announce a brand new product.
          </span>
        </p>
      </div>
      <div
        className="order-3 mt-2 w-full shrink-0 sm:order-2 sm:mt-0 sm:w-auto"
      >
        <a
          className="flex items-center justify-center rounded-md border border-transparent bg-white py-2 px-4 text-sm font-medium text-indigo-600 shadow-sm hover:bg-indigo-50"
          href="@/components/banner/Banner#"
        >
          Learn more
        </a>
      </div>
      <div
        className="order-2 shrink-0 sm:order-3 sm:ml-3"
      >
        <button
          className="-mr-1 flex rounded-md p-2 hover:bg-indigo-500 focus:outline-none focus:ring-2 focus:ring-white sm:-mr-2"
          type="button"
        >
          <span
            className="sr-only"
          >
            Dismiss
          </span>
          <svg
            aria-hidden="true"
            className="h-6 w-6 text-white"
            fill="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clipRule="evenodd"
              d="M5.47 5.47a.75.75 0 011.06 0L12 10.94l5.47-5.47a.75.75 0 111.06 1.06L13.06 12l5.47 5.47a.75.75 0 11-1.06 1.06L12 13.06l-5.47 5.47a.75.75 0 01-1.06-1.06L10.94 12 5.47 6.53a.75.75 0 010-1.06z"
              fillRule="evenodd"
            />
          </svg>
        </button>
      </div>
    </div>
  </div>
</div>
`;
